src/pages/tasks/common/hooks.tsx

```diff
@@ -367,7 +367,7 @@ export function useActions() {
       !task.invoice_id && (
         <DropdownElement
           onClick={() => invoiceTask([task])}
-          icon={<Icon element={MdTextSnippet} />}
+          icon={<Icon element={BiPlusCircle} />}
```

Comment:
Why? We are trying to stick either to Feather Icons or Material for the UI.

Reply:
Actually, I thought the new icon was better for UX, but as we discussed today on Slack for type of icons that we have to use and also after reviewing I think `MdTextSnippet` could be even better for UX. So I went back to the old version. Let me know your thoughts.